content/docs/identity-providers/google.mdx

```diff
@@ -90,38 +90,40 @@ IDP_CLIENT_SECRET="xxxxxx"
 
 ## Getting Groups
 
-<Tabs queryString="get-groups">
-<TabItem value="custom-claim" label="Custom Claim (Open Source)">
+### Custom claims
 
-Unfortunately, Google does not yet support getting groups data using a custom claim. Groups must be loaded by using a plugin to fetch directory information (see Enterprise's Directory Sync).
+Unfortunately, Google doesn't support getting groups data with a custom claim. To get and use groups data from your Google directory, you must conduct a directory sync in the Enterprise Console.
```

Comment:
I think I've mentioned this before, but "conduct a directory sync" makes this sound like a one-time event. I'd suggest we phrase this more like "enable directory sync" to help reflect that directory sync is a recurring process.

Reply:
I think this stems from a misunderstanding on my part of how directory sync works. I thought it was a one-time sync, I'll update. Thanks for catching that.